supervisor/supervisor.mk

```diff
@@ -35,6 +35,9 @@ CFLAGS += -DQSPI_FLASH_FILESYSTEM=$(QSPI_FLASH_FILESYSTEM)
 SPI_FLASH_FILESYSTEM ?= 0
 CFLAGS += -DSPI_FLASH_FILESYSTEM=$(SPI_FLASH_FILESYSTEM)
 
+DISABLE_FILESYSTEM ?= 0
```

Comment:
I think it makes sense to move the `ifeq ($(DISABLE_FILESYSTEM),1)` checks from above to below this. That way it's always defined when doing the check.

Reply:
Definitely, you are right. Nice work!

Here is the fix:
https://github.com/EmergReanimator/circuitpython-dut/tree/3c588708166a218116620b77349c2af5a4b04913